cli/test/lib/commands/transform.test.ts

```diff
@@ -46,6 +47,20 @@ describe('@cube-creator/cli/lib/commands/transform', function () {
       expect(shape).to.be.true
     })
 
+    it('removes previous errors from dataset', async () => {
+      const datasetGraph = projectNs.dataset
+
+      const hasPreviousErrors = ASK`
+        <dataset> schema:error <dataset#non-unique-observations> .
+        <dataset> schema:error <dataset#missing-values-error> ; ${schema.description} "Test message" .
+      `
+        .FROM(datasetGraph)
+        .prologue`BASE ${projectNs()}`
```

Comment:
Just curious, what is `prologue`?

Reply:
That's what SPARQL calls in the spec the part with prefixes and `BASE`